designcompose/src/main/java/com/android/designcompose/DesignView.kt

```diff
@@ -778,6 +778,16 @@ internal val LocalDesignIsRootContext = compositionLocalOf { DesignIsRoot(true)
 // Current customization context that contains all customizations passed through from any ancestor
 val LocalCustomizationContext = compositionLocalOf { CustomizationContext() }
 
+// Current document override ID that can be used to override the document ID specified from the
+// @DesignDoc annotation
+internal val LocalDocOverrideContext = compositionLocalOf { String() }
+
+// Public function to set the document override ID
+@Composable
```

Comment:
Add the Discouraged annotation:
https://developer.android.com/reference/androidx/annotation/Discouraged

Reply:
Done